test/kpod_run.bats

```diff
@@ -128,10 +128,4 @@ IMAGE="docker.io/library/fedora:latest"
     run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} run -v ${MOUNT_PATH}:/run/test:ro ${FEDORA_MINIMAL} cat /proc/self/mountinfo | grep '${MOUNT_PATH} /run/test ro,relatime'"
     echo $output
     [ "$status" -eq 0 ]
-    #run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} run -v ${MOUNT_PATH}:/run/test:shared ${FEDORA_MINIMAL} cat /proc/self/mountinfo | grep '${MOUNT_PATH} /run/test rw,relatime shared:'"
```

Comment:
No tests?

Reply:
The tests were passing everywhere else but failing on Travis and I couldn't figure out why. Added them back in